index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+    <link rel="stylesheet" href="tributePage.css">
+</head>
+<body>
+    <header class="Alex">
+        <h1>Dr. Jane Goodall</h1>
+        <p>The Woman who saved billions of life</p>
+    </header>
+    <main id="main">
+        <div id="img-div">
+        <img id="image" src="https://upload.wikimedia.org/wikipedia/commons/thumb/8/84/JaneGoodallOct10.jpg/330px-JaneGoodallOct10.jpg" alt="">
+        <p>Dr. Jane Goodall, trians biologists in italy on how to increase 
+            wheat yields - part of her life-long war on hunger</p>
+    </div>
```

Comment:
Please you can use the section tag instead of div, it gives more semantic meaning to your codebase.